src/Entity/Summary.php

```diff
@@ -137,7 +137,7 @@ class Summary
     /**
      * @var Skill[]|Collection
      *
-     * @ORM\OneToMany(targetEntity="AppBundle\Entity\MemberSummary\Skill", mappedBy="summary", indexBy="id", cascade={"all"}, orphanRemoval=true)
```

Comment:
Yes this is, we want to handle keys for each collection to allow additions of new items, we don't want to use ids since we could not increment the count to insert elements then.